src/renderer/renderer.js

```diff
@@ -148,7 +148,6 @@ export class Renderer extends GameComponent {
   
   // DRAW SCENE
   renderFrame(remakeMesh){
```

Comment:
This isn't even called with an argument any more so the parameter is not needed at all